channels/kw.m3u

```diff
@@ -1,26 +1,30 @@
 #EXTINF:-1 tvg-id="AlShahed.kw" tvg-name="Al-Shahed" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://alshahed.tv/wp-content/themes/twentyfifteen/asset/img/logo.png" group-title="",Al-Shahed [Offline]
 https://easybroadcast.akamaized.net/abr_live/Al-Shahed/live/Al-Shahed_720p/chunks.m3u8
-#EXTINF:-1 tvg-id="Alkasssport1.kw" tvg-name="Alkass sport 1" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Alkass sport 1 (304p)
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass1
-#EXTINF:-1 tvg-id="AlKassSports2.kw" tvg-name="Al Kass Sports 2" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Al Kass Sports 2 [Offline]
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass2
-#EXTINF:-1 tvg-id="AlKassSports3.kw" tvg-name="Al Kass Sports 3" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Al Kass Sports 3 [Offline]
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass3
-#EXTINF:-1 tvg-id="AlKassSports.kw" tvg-name="Alkass sport 4" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Al Kass Sports 4 [Offline]
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass4
-#EXTINF:-1 tvg-id="AlKassSports5.kw" tvg-name="Alkass sport 5" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Al Kass Sports 5 [Offline]
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass5
+#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV
+http://stream02.fasttelco.net/4/pub/asset/28/streams.m3u8?v=1053886998
+#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV (1080p)
+https://stream02.fasttelco.net/4/pub/asset/28/streams.m3u8
+#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV
+https://stream02.fasttelco.net/live/alrai.stream/chunklist.m3u8
+#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV
+https://stream02.fasttelco.net/live/alrai.stream/chunklist_w2023460738.m3u8
+#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV
+https://stream02.fasttelco.net/live/alrai.stream/chunklist_w498917617.m3u8?v=2957056943
+#EXTINF:-1 tvg-id="ATV.kw" tvg-name="ATV" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.atvkuwait.com/img/logo/atv_logo.png" group-title="",ATV (360p)
+https://gulfsat.cdn.easybroadcast.fr/live/Aladalah_abr/playlist.m3u8
+#EXTINF:-1 tvg-id="ATV.kw" tvg-name="ATV" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://www.atvkuwait.com/img/logo/atv_logo.png" group-title="",ATV
+rtmp://168.187.49.14:1935/live/atv?v=
```

Comment:
Not sure what happened there, this chunk contains questionable and non-adaptive links that we have removed before